Cannot find module 'core-js/modules/es.array.iterator'

I don't know if it's just me, but after updating to `v1.1.14`, running `:Format` gives me this error and the file is not formatted:

```
## Output channel: prettier

[Error - 5/27/2020, 12:15:04 PM:] Cannot find module 'core-js/modules/es.array.iterator'
```

I fixed it by installing `core-js`, but I'm not sure if that's the proper solution (I noticed it was removed in the [latest commit](https://github.com/neoclide/coc-prettier/commit/cb72171e112313bd58545ebecdc390a3b2647a3b#diff-8ee2343978836a779dc9f8d6b794c3b2L1086), since it's not a `prettier-eslint` requirement anymore). If it is, I'd be happy to send a PR.

--- a/package.json
+++ b/package.json
@@ -236,6 +236,7 @@
     "webpack-cli": "^3.3.11"
   },
   "dependencies": {
+    "core-js": "^3.6.5",
     "prettier": "^2.0.5",
     "prettier-eslint": "^10.1.1",
     "prettier-stylelint": "^0.4.2",

